### ISSUE TITLE
NIMROD: help recomission jaws

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#5185]
As a NIMROD scientist I would like help finding out where my jaws are.

As a result of https://github.com/ISISComputingGroup/IBEX/issues/5181 :
- The galil power supply has been changed, which means that the voltage reading from the jaws may be scaled differently than before.
- The jaws have been moved, so we cannot rely on the old position being equal to the new position and redefining
- We cannot see the jaws as they are buried under shielding

The scientist plans to find out where their jaws are using neutrons and comparing against spectra from last cycle. They would like someone from the IBEX team to be present at 9:30 on monday (10/02) to help them to redefine positions on each of the jaw blades (6 jaw sets so 24 axes).

**Comment by John-Holt-Tessella** [comment]
@Tom-Willemsen @ChrisColeExpControl did this this morning. The scientists will let us know if it looks weird
